Adding a note on dvc metrics diff --targets with revisions usage (#1573)
iterative/dvc.org: content/docs/command-reference/metrics/diff.md

## Options

- `--targets <paths>` - limit command scope to these metric files. Using -R,
  directories to search metric files in can also be given. When specifying
  multiple `--targets` before `revisions`, you may use `--` after this option's
  arguments, e.g.:

  ```dvc

>   $ dvc metrics diff --targets t1.json t2.csv -- HEAD v1

Reviewer: metrics diff doesn't support CSV. Please think through anything copy pasted among commands. See https://dvc.org/doc/command-reference/metrics#supported-file-formats

Author: Thanks! I will now verify everything before pushing changes.

## Options

- `--targets <paths>` - limit command scope to these metric files. Using -R,
  directories to search metric files in can also be given. When specifying
  arguments for `--targets` before `revisions`, you may use `--` after this
  option's arguments, e.g.:

  ```dvc

>   $ dvc metrics diff --targets t1.json t2.json -- HEAD v1

Reviewer: > Metrics can be organized as tree hierarchies in JSON or YAML files

Form https://dvc.org/doc/command-reference/metrics#supported-file-formats

```suggestion
  $ dvc metrics diff --targets t1.json t2.yaml -- HEAD v1
```

  ```

>   Without `--`, this statement will show an error as argument parser confuses
>   `revisions` as arguments for `--targets` option. Alternatively, you can also
>   run above statement as:

Reviewer: Let's just avoid this sentence and change "may" for "must" or "should" in the previous one, please. Also, "the" is missing again 😞 

```suggestion
  Alternatively, you can also run the above statement as:
```

Same for plots diff, tnx
